src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -1593,7 +1593,9 @@ To create/register a plugin, you have to :
     #undef USES_C018 // LoRa TTN - RN2483/RN2903
   #endif
   #ifdef USE_TRIGONOMETRIC_FUNCTIONS_RULES
-    #undef USE_TRIGONOMETRIC_FUNCTIONS_RULES
+    #ifndef SIZE_1M
+      #undef USE_TRIGONOMETRIC_FUNCTIONS_RULES
```

Comment:
This is becoming quite hard to follow to have exceptions in exceptions.
Not sure right now what would be a better solution, but this is bound to go wrong at some time.

Reply:
I tried to simplify things, but of course that won't work 😆 though I expect it is now more comprehendible.